sdk/src/test/scala/com/akkaserverless/javasdk/impl/valueentity/AnnotationBasedValueEntitySupportSpec.scala

```diff
@@ -1,309 +0,0 @@
-/*
- * Copyright 2021 Lightbend Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.akkaserverless.javasdk.impl.valueentity
-
-import com.akkaserverless.javasdk.impl.reply.MessageReplyImpl
-import com.akkaserverless.javasdk.valueentity._
-import com.akkaserverless.javasdk.impl.{AnySupport, ResolvedServiceMethod, ResolvedType}
-import com.akkaserverless.javasdk.{Reply, EntityContext => _, _}
-import com.example.valueentity.shoppingcart.ShoppingCartApi
-import com.google.protobuf.any.{Any => ScalaPbAny}
-import com.google.protobuf.{ByteString, Any => JavaPbAny}
-import org.scalatest.wordspec.AnyWordSpec
-import org.scalatest.matchers.should.Matchers
-import java.util.Optional
-
-import scala.compat.java8.OptionConverters._
-
-import com.akkaserverless.javasdk.lowlevel.ValueEntityHandler
-
-class AnnotationBasedValueEntitySupportSpec extends AnyWordSpec with Matchers {
```

Comment:
I previously thought we'd like to keep this to use it as a checklist to see if our explicit API indeed supports all the thing the reflection/annotation-based API supported before, but looking closer all tests seem pretty annotation-specific and not so relevant for the explicit API. So :+1:

Reply:
yes, I will revert